ivy/functional/backends/paddle/random.py

```diff
@@ -133,4 +133,11 @@ def shuffle(
     seed: Optional[int] = None,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    if seed:
+        _ = paddle.seed(seed)
+    # Use numpy's permutation function to shuffle indices
+    indices = paddle.to_tensor(np.random.permutation(x.shape[0]), dtype='int64')
```

Comment:
Hi, the implementation looks good to me, please use only paddle functions in paddle backend, there is a function called randperm that can be used instead
also it doesn't support native out so remove the last line

Reply:
yes done changes according to given feedback remove last line and uses randperm function to it